### PR TITLE
Enable Golang implementation of PKI globally

### PR DESCRIPTION
**Pull request by vkhoroz**
We have already tested this implementation on Windows in the field.
Also tested it on Linux using a local build.

Signed-off-by: Volodymyr Khoroz <volodymyr.khoroz@foundries.io>

### CODE REVIEW

**Comment by vkhoroz** [comment]
I actually tested it with MEDS and it works well.

**Comment by detsch** [comment]
Just a reminder that HSM support was not added to the Golang implementation.
Other than that, LGTM.

**Review by detsch — APPROVED** [review]

File: x509/golang.go [hunk 1]
```diff
@@ -1,4 +1,6 @@
-//go:build windows
+//go:build go1.15 && !bashpki
```
Comment (doanac) [inline x509/golang.go:1]: so `go1.15` means greater than or equal to go1.15?

Comment (doanac) [reply x509/golang.go:1]: i.e. our go.mod is pinned on go1.19 already

Comment (vkhoroz) [reply x509/golang.go:1]: Yes and Yes

Maybe this is redundant, and we only need the `bashpki` flag.
In the initial version I thought to have no `bashpki`, and needed some flag to disable the bash implementation.

**Comment by vkhoroz** [comment]
Closing until I add the HSM support to Golang implementation.
Then I will reopen.

**Pull request closed without merging**